### ISSUE TITLE
Crear Datasets (tipo Bunch) con labels numéricas para regresión

### ISSUE THREAD

**Issue by AdrianArnaiz** [AdrianArnaiz/TFG-Neurodegenerative-Disease-Detection#51]
Adición de labels numéricas.

Como los datasets fueron guardados en formato numpy, modificaremos para guardarlos en formato Bunch (tipo de diccionario) al igual que importa los datos sklearn.

Para ello utilizaremos los datos de Disvocice + Edad y Sexo y les añadiremos UPDRS, UPDRS_SPEECH y HY, 3 escalas de medición de severidad del Parkinson. También añadiremos esas labels a los Embeddings de VGGish.

Como en algunos conjuntos de datos originales había Nan y fueron eliminados, no sabemos a qué audio se corresponde cada instancia, por lo que habrá que volver a sacar en algunos casos las características.
